lib/HalModuleParser.js

```diff
@@ -245,7 +245,7 @@ HalModuleParser.prototype = {
 		var mightBeSystem = (sysModuleStartAddy.indexOf("80") === 0);
 		var isCore = ((userModuleStartAddy.indexOf("20") === 0) && (coreModuleStartAddy === "8005000"));
 		var mightBeBluz = (bluzModulesPlatformID == 103 && bluzModuleStartAddy < 262144);
-		var isMesh = (meshModule.platformID >= 12 && meshModule.platformID <= 30) && (meshModule.moduleStartAddy === '30000' || meshModule.moduleStartAddy === 'f4000') &&
+		var isMesh = (meshModule.platformID >= 12 && meshModule.platformID <= 25) && (meshModule.moduleStartAddy === '30000' || meshModule.moduleStartAddy === 'f4000') &&
```

Comment:
You can rename this to `isGen3`. The Asset Tracker must qualify for this check since the module format is the same as other Gen 3 platforms.

Reply:
✅ done in https://github.com/particle-iot/binary-version-reader/pull/33/commits/94a4bff7bf015d34d1e0f4f729a6808d078b5d19